File: Processview_android/app/src/main/java/com/im/daeseong/processview/MainActivity.java
package com.im.daeseong.processview;

import androidx.appcompat.app.AppCompatActivity;//import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.util.Log;
import java.util.List;

public class MainActivity extends AppCompatActivity {

    private static final String TAG = MainActivity.class.getSimpleName();

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        List<String> runService = CProcessInfo.getRunningServicesName(MainActivity.this);
        for (String packageName : runService) {
            Log.e(TAG, "runService:" + packageName);
        }

        List<String> runTop = CProcessInfo.getRunningTopPackageName(MainActivity.this);
        for (String packageName : runTop) {
            Log.e(TAG, "runTop:" + packageName);
        }

        List<String> currentTask = CProcessInfo.getCurrentTaskPackageName(MainActivity.this);
        for (String packageName : currentTask) {
            Log.e(TAG, "CurrentTask:" + packageName);
        }

        List<String> allPackage = CProcessInfo.getAllPackageNames(MainActivity.this);
        for (String packageName : allPackage) {
            Log.e(TAG, "allPackage:" + packageName);
        }
    }
}

File: processview(kotlin)/app/src/main/java/com/daeseong/processview/MainActivity.kt
package com.daeseong.processview

import android.os.Bundle
import android.util.Log
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val runService = CProcessInfo.getRunningServicesName(this)
        for (i in runService!!.indices) {
            Log.e("tag", "runService:" + runService[i])
        }

        val runTop = CProcessInfo.getRunningTopPackageName(this)
        for (i in runTop!!.indices) {
            Log.e("tag", "runTop:" + runTop[i])
        }

        val currentTask = CProcessInfo.getCurrentTaskPackageName(this)
        for (i in currentTask!!.indices) {
            Log.e("tag", "CurrentTask:" + currentTask[i])
        }

        val allPackage = CProcessInfo.getAllPackageNames(this)
        for (i in allPackage!!.indices) {
            Log.e("tag", "allPackage:" + allPackage[i])
        }
    }
}
